apps/api/src/app/events/services/sms-service/handlers/index.ts

```diff
@@ -6,3 +6,4 @@ export * from './sms77.handler';
 export * from './termii.handler';
 export * from './gupshup.handler';
 export * from './firetext.handler';
+export * from './burstsms.handler';
```

Comment:
```suggestion
export * from './burst-sms.handler';
```
In order to the previous comment. Apologies as I missed it.